integrations/access/msteams/app.go

```diff
@@ -471,6 +527,14 @@ func (a *App) getMessageRecipients(ctx context.Context, req types.AccessRequest)
 	// This can happen if this set contains the channel `C` and the email for channel `C`.
 	recipientSet := stringset.New()
 
+	accessRuleRecipients := a.accessMonitoringRules.RecipientsFromAccessMonitoringRules(ctx, req)
+	accessRuleRecipients.ForEach(func(r common.Recipient) {
+		recipientSet.Add(r.Name)
+	})
+	if recipientSet.Len() != 0 {
+		return recipientSet.ToSlice()
```

Comment:
This is to follow the behavior used in the other plugins.

Reply:
The standard for the other plugins has been that the access monitoring rules replace existing recipients and are not additive. We could change that here if you think it makes more sense, it would mean that it would have different behavior to the other plugins unless we are willing to make changes to the behavior of all of them